README.md

```diff
@@ -5,6 +5,16 @@
 -----
 [CircuitVerse](https://circuitverse.org) is a free, open-source platform which allows users to construct digital logic circuits online. We also offer the [Interactive Book](https://learn.circuitverse.org) which teaches users on the fundamentals of modern, digital circuits. Please also see our [documentation](https://docs.circuitverse.org) or [GitHub Wiki](https://github.com/CircuitVerse/CircuitVerse/wiki/).
 
+## Getting Started with CircuitVerse 📹
+To help new contributors get started, we've created a video guide titled **"Getting Started with CircuitVerse"**. This video covers:
+- Introduction to CircuitVerse and its technical structure.
+- Step-by-step setup using GitHub Codespaces.
+- A detailed walkthrough of the codebase directory structure and key files.
```

Comment:
```suggestion
- A high level walkthrough of the codebase directory structure and key files.
```

we haven't done detailed walk though

Reply:
done, also if needed would love doing a detailed version, would just need to know what it would constitute.